generators/writing/templates/src/seed-data.ejs

```diff
@@ -22,10 +25,10 @@ let services = (readJsonFileSync(join(__dirname, '../feathers-gen-specs.json'))
 <%- insertFragment('init') %>
 
 <%- tplExport('async function (app) {', 'async function (app: App) {') %>
-  const ifDbChangesAllowed = areDbChangesAllowed(app.get("tests"))<%- sc %>
+  const ifDbChangesAllowed = areDbChangesAllowed(app.get('tests'))<%- sc %>
   <%- insertFragment('func_init') %>
-  if (!ifSeedServices) return<%- sc %>
-  if (!ifDbChangesAllowed) return<%- sc %>
+  if (!ifSeedServices) { return<%- sc -%> }
+  if (!ifDbChangesAllowed) { return<%- sc -%> }
```

Comment:
the current tslint.json allows one-line unbraced if statements, so there should be no need to change this